vision:
  ros__parameters:
    publish_rate: 30 # in Hz

    camera:
      width_capture: 640
      height_capture: 480
      framerate: 30

    mqtt:
      port: 18789
      broker: "driver.cloudmqtt.com"

    precision_land:
      kp: 0.1
      radius: 10 # in pixels
      height: 0.2

    object_detection:
      confidence_threshold: 0.25
      iou_threshold: 0.45
      model_name: "crow-int8.tflite"